test/aqua/runDeployedDeals.aqua

```diff
@@ -0,0 +1,31 @@
+aqua Main
+
+import "@fluencelabs/aqua-lib/builtin.aqua"
+import "@fluencelabs/registry/subnetwork.aqua"
+import Registry from "@fluencelabs/registry/registry-service.aqua"
+import "@fluencelabs/spell/spell_service.aqua"
+import Ipfs from "@fluencelabs/aqua-ipfs/ipfs.aqua"
+
+import "workers.aqua"
+import "services.aqua"
+
+export status
+
+data Answer:
+    answer: string
+    peer: string
+
+func status() -> *Answer:
+    workersInfo <- getWorkersInfo()
+    dealId = workersInfo.deals.defaultWorker.dealId
+    answers: *Answer
+    on HOST_PEER_ID:
```

Comment:
you can remove `on HOST_PEER_ID` because it's already done in [resolveSubnetwork](https://github.com/fluencelabs/registry/blob/db68f9bf4516cea53500fefcdf329fb9a5a678b2/aqua/subnetwork.aqua#L153)